src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java

```diff
@@ -179,7 +179,7 @@ private PaymentIntention acceptedPaymentIntention(ResponseAcceptation responseAc
                     .getCourtDetermination().orElseThrow(IllegalStateException::new)
                     .getCourtDecision();
             default:
-                throw new IllegalStateException("Invalid determination decision type " + determinationDecisionType);
+                throw new IllegalStateException("Invalid determination decision type " + decisionType);
```

Comment:
Shouldn't comment get updated as well in case class rename?

Reply:
agree